code/game/gamemodes/blob/blobs/core.dm

```diff
@@ -63,7 +63,8 @@
 
 /obj/effect/blob/core/Life()
 	if(!overmind)
-		create_overmind()
+		if(!qdeleted(src))
```

Comment:
I'm hoping wildly that this fixes an unreported bug where the core fucking makes an overmind in nullspace(obviously bad!)